korman/plasma_magic.py

```diff
@@ -0,0 +1,225 @@
+#    This file is part of Korman.
+#
+#    Korman is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    Korman is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with Korman.  If not, see <http://www.gnu.org/licenses/>.
+
+very_very_special_python = """
+from Plasma import *
+from PlasmaTypes import *
+
+globals()["{age_name}"] = type("{age_name}", (ptResponder,), dict())
+"""
+
+very_very_special_sdl = """
+#==============================================================
+# This VeryVerySpecial SDL File was automatically generated
+# by Korman. Have a nice day!
+#
+# READ: When modifying an SDL record, do *not* modify the
+#       existing record. You must copy and paste a new version
+#       below the current one and make your changes there.
+#==============================================================
+
+STATEDESC {age_name}
+{{
+    VERSION 0
+}}
+
+"""
+
+# Copypasta (with small fixes for str.format) of glue.py from CWE's moul-scripts
+plasma_python_glue = """
+# glue code in python for the glue code in C++
+# This assumes that this will be loaded into the module that we are trying to do
+# with an execfile('.\\python\\system\\glue.py') at the end of the module (after everything is defined)
```

Comment:
I see that this is an error in the original file, but the path is incorrect.  It's currently found in [Python/plasma/glue.py](https://github.com/H-uru/moul-scripts/blob/master/Python/plasma/glue.py).

Reply:
I should probably just remove the comments entirely... The code is compiled and the comments trashed :wink: